src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -193,6 +193,8 @@ class BaseCCPPVisitor : public ASR::BaseVisitor<Struct>
 
     void visit_TranslationUnit(const ASR::TranslationUnit_t &x) {
         global_scope = x.m_global_scope;
+        SymbolTable current_scope_copy = current_scope;
```

Comment:
Shouldn't you do:
```suggestion
        SymbolTable *current_scope_copy = current_scope;
```
?

Reply:
Yes, thanks